extensions/schema-compare/src/schemaCompareMainWindow.ts

```diff
@@ -913,7 +913,7 @@ export class SchemaCompareMainWindow {
 
 		this.selectTargetButton.onDidClick(async () => {
 			TelemetryReporter.sendActionEvent(TelemetryViews.SchemaCompareMainWindow, 'SchemaCompareSelectTarget');
-			this.schemaCompareDialog = new SchemaCompareDialog(this);
+			this.schemaCompareDialog = new SchemaCompareDialog(this, undefined, this.extensionContext);
 			this.promise = await this.schemaCompareDialog.openDialog();
```

Comment:
check if you need to update or add tests for this - the PR build seems to fail please take a look.

Reply:
Yes, the test need to be updated. I will fix them.